x-pack/plugins/alerting/server/saved_objects/index.ts

```diff
@@ -61,7 +61,6 @@ export type AlertAttributesExcludedFromAADType =
   | 'updatedAt'
   | 'executionStatus'
   | 'monitoring'
-  | 'snoozeEndTime'
```

Comment:
Removing `extends SavedObjectAttributes` from revealed that this property is not valid any more

Reply:
We need to keep this due to the comment above: `field removed in 8.2, but must be retained in case an rule created/updated in 8.2 is being migrated`

Reply:
Thanks @ymao1! I will keep it.